### PR TITLE
Update jp.m3u

### PR DESCRIPTION
**Pull request by Dum4G**

### CODE REVIEW

File: channels/jp.m3u [hunk 1]
```diff
@@ -1,12 +1,18 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese" tvg-logo="" group-title="Animation",Animax
-https://redlabmcdn.s.llnwi.net/jp01/bs14/tracks-v1a1/mono.m3u8
 #EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Indonesian" tvg-logo="" group-title="Animation",Animax Japan (Indonesian Subs)
 http://210.210.155.35/dr9445/h/h02/01.m3u8
```
Comment (LaneSh4d0w) [inline channels/jp.m3u:3]: Those links don't seem to work for me at the same instance.

File: channels/jp.m3u [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
 #EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Indonesian" tvg-logo="" group-title="Animation",Animax Japan (Indonesian Subs)
 http://210.210.155.35/dr9445/h/h144/01.m3u8
+#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Indonesian" tvg-logo="" group-title="Animation",Animax Japan (Indonesian Subs)
+http://185.180.221.194:8278/streams/d/Animax/playlist.m3u8
+#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Indonesian" tvg-logo="" group-title="Animation",Animax Japan (Indonesian Subs)
+#EXTVLCOPT:http-user-agent="Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36"
+https://tv.mbed.cc/u/chunklist.m3u8?id=animax&bandwidth=default&sync=0
```
Comment (LaneSh4d0w) [inline channels/jp.m3u:10]:
```suggestion
https://tv.mbed.cc/u/master.m3u8?id=animax&bandwidth=default&sync=0
```

by the way : https://tv.mbed.cc/list.txt ;)

**Comment by LaneSh4d0w** [comment]
@Dum4G You did something very interesting...

#EXTVLCOPT:http-user-agent="Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36"

works on the fptplay link you gave.

Try to add it on the other links to see if this works.

File: channels/jp.m3u [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
 #EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Vietnamese" tvg-logo="" group-title="Animation",Animax Japan (Vietnamese Subs)
-https://livecdn.fptplay.net/hda3/animaxport_2000.stream/.m3u8
+#EXTVLCOPT:http-user-agent="Mozilla/5.0 Macintosh; Intel Mac OS X 10_14_5 AppleWebKit/537.36 KHTML, like Gecko Chrome/76.0.3809.25 Safari/537.36"
+https://livecdn.fptplay.net/hda3/animaxport_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Thai" tvg-logo="" group-title="Animation",Animax Japan (Thai Subs)
+http://27.254.130.62/feed/LC38/tracks-v1a1/mono.m3u8
```
Comment (LaneSh4d0w) [inline channels/jp.m3u:15]:
```suggestion
http://27.254.130.62/feed/LC38/playlist.m3u8
```

File: channels/jp.m3u [hunk 4]
```diff
@@ -59,12 +65,10 @@ https://nhk.mov3.co/hls/nhk.m3u8
...
 https://nhkwlive-ojp.akamaized.net/hls/live/2003459/nhkwlive-ojp-en/index.m3u8
-#EXTINF:-1 tvg-id="NHKWorldJapan.jp" tvg-name="NHK World Japan" tvg-country="JP" tvg-language="" tvg-logo="" group-title="",NHK World Japan
-https://nhkwlive-xjp.akamaized.net/hls/live/2003458/nhkwlive-xjp-en/index_600k.m3u8
 #EXTINF:-1 tvg-id="NHKWorldJapan.jp" tvg-name="NHK World Japan" tvg-country="JP" tvg-language="Japanese" tvg-logo="https://i.imgur.com/SQISXoD.jpg" group-title="News",NHK World Japan (720p)
 https://nhkworld.webcdn.stream.ne.jp/www11/nhkworld-tv/global/2003458/live.m3u8
-#EXTINF:-1 tvg-id="NHKWorldPremium.jp" tvg-name="NHK World Premium" tvg-country="JP" tvg-language="" tvg-logo="https://nhkworldpremium.com/img/common/logo.png" group-title="",NHK World Premium
-https://liveanevia.mncnow.id/live/eds/NHKWorldPremium/sa_dash_vmx/NHKWorldPremium.mpd
+#EXTINF:-1 tvg-id="NHKWorldJapan.jp" tvg-name="NHK World Japan" tvg-country="JP" tvg-language="Japanese" tvg-logo="https://i.imgur.com/SQISXoD.jpg" group-title="News",NHK World Japan (720p)
+https://b-nhkwlive-xjp.webcdn.stream.ne.jp/hls/live/2003458-b/nhkwlive-xjp-en/index_1M.m3u8
```
Comment (LaneSh4d0w) [inline channels/jp.m3u:71]:
```suggestion
https://b-nhkwlive-xjp.webcdn.stream.ne.jp/hls/live/2003458-b/nhkwlive-xjp-en/index.m3u8
```

https://nhkwlive-xjp.webcdn.stream.ne.jp/hls/live/2003458/nhkwlive-xjp-en/index.m3u8

is the main stream.

File: channels/jp.m3u (hunk 1 above)
Comment (Dum4G) [reply channels/jp.m3u:3]: Shows Aniplus for now tho
image

**Comment by Dum4G** [comment]
> Try to add it on the other links to see if this works.

Nah, original Animax is lost, not even VPN helps

**Comment by LaneSh4d0w** [comment]
> 
> 
> > Try to add it on the other links to see if this works.
> 
> Nah, original Animax is lost, not even VPN helps

Wanted to mention other Indonesian links that used this host.

File: channels/jp.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/jp.m3u:3]: Very interesting... let me try again on my part, because I can't even connect to host with what I have.

Comment (LaneSh4d0w) [reply channels/jp.m3u:3]: Found out : it's geo-locked.
vlcsnap-2021-07-13-00h26m43s718

Comment (Dum4G) [reply channels/jp.m3u:3]: Interesting. Indonesian stream that I can play without VPN

Comment (LaneSh4d0w) [reply channels/jp.m3u:3]: Maybe it's locked for the Asian continent / diaspora... I don't know. With VPN disabled just after, it locks back.

**Review by BellezaEmporium — APPROVED** [review]
